docs/Training-ML-Agents.md

```diff
@@ -8,17 +8,17 @@ Use the Python program, `learn.py` to train your agents. This program can be fou
 
 For a broader overview of reinforcement learning, imitation learning and the ML-Agents training process, see [ML-Agents Overview](ML-Agents-Overview.md).
```

Comment:
Maybe this guide should disappear as well

Reply:
Yes if we move this into the basic guides, it will be too detailed.